packages/node_modules/@node-red/editor-client/src/js/ui/common/treeList.js

```diff
@@ -709,6 +722,16 @@
                 return this._data;
             }
         },
+        item: function(newItem, select) {
```

Comment:
What is this function for? I can't see it being part of this fix and there's no mention of it in the PR.

Reply:
Sorry, this has nothing to do with it. It is a new feature that allows you to add an item at level 0. It was added later in the commit, and I reflected it in the pull request by mistake. I remove it